custom_components/senseme/__init__.py

```diff
@@ -2,9 +2,9 @@
 import asyncio
 import logging
 
-from aiosenseme import SensemeDevice
+from .aiosenseme.device import SensemeDevice
```

Comment:
These changes will be removed once mikelawrence/aiosenseme#11 is merged

Reply:
You can install local copy of the aiosenseme library and you won't have to make changes like those in the `__init__.py` file.
```
pip install -e directory_of_aiosenseme_clone
```